common/src/main/java/org/apache/celeborn/common/network/client/TransportClient.java

```diff
@@ -175,6 +175,19 @@ public ChannelFuture pushData(
     return channelFuture;
   }
 
+  public ChannelFuture pushDataWithCompleteCallback(
```

Comment:
This method can be merged with previous method because listener is different.

Reply:
ok, merge common code